srcpkgs/comfy/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'comfy'
+pkgname=comfy
+version=0.5.4
+revision=1
+build_style=cargo
+short_desc="Flexible command script manager / runner written in Rust"
+maintainer="daCoUSB <usbdaco@protonmail.com>"
+license="MIT"
+homepage="https://github.com/dacousb/comfy"
+distfiles="https://github.com/dacousb/comfy/archive/v${version}.tar.gz"
```

Comment:
Since the homepage is contained in distfiles, `distfiles="${homepage}/archive/v${version}.tar.gz"` might look better.